drivers/media/i2c/imx296.c

```diff
@@ -539,6 +543,7 @@ static int imx296_stream_off(struct imx296 *sensor)
 	int ret = 0;
 
 	imx296_write(sensor, IMX296_CTRL0A, IMX296_CTRL0A_XMSTA, &ret);
+	usleep_range(IMX296_STREAM_OFF_DELAY, 2*IMX296_STREAM_OFF_DELAY);
```

Comment:
"Sensor Setting Flow (Sensor Master Mode)"  lists the correct sequence as setting STANDBY=1, wait >1ms, and then XMSTA=1.

Reply:
I recall these delays directly from Sony as things were just plain unreliable without them.  I'll dig out the emails (if I still have them) to verify.

Reply:
My comment was more that the datasheet states ```IMX296_CTRL00, IMX296_CTRL00_STANDBY``` first, then wait >1ms, then ```IMX296_CTRL0A, IMX296_CTRL0A_XMSTA```. The register writes are currently reversed.

Reply:
Yes, I did notice that as well.  Both my driver and Laurent's driver use the given sequence (don't ask me why, it's been too long), and I kept it unchanged because it does seem to work.

Reply:
OK, we're not using Master/Slave mode at present, so it may be irrelevant. One to bear in mind should we introduce it.

Reply:
In time we will definitely add support for this!